topics/imaging/tutorials/2D-spot-detection/tutorial.md

```diff
@@ -0,0 +1,235 @@
+Therefore, for improved visibility the intensity histogram of the image can be normalized first.
+
+We will normalize the histogram to improve the contrast. 
+We do this using a [Contrast Limited Adaptive Histogram Equalization (CLAHE)](https://en.wikipedia.org/wiki/Adaptive_histogram_equalization) approach.
+
+> <hands-on-title>Normalize Histogram</hands-on-title>
+>
+> 1. {% tool [Perform histogram equalization](toolshed.g2.bx.psu.edu/repos/imgteam/2d_histogram_equalization/ip_histogram_equalization/0.18.1+galaxy0) %} with the following parameters to normalize the histogram of the image:
+>    - {% icon param-file %} *"Input image"*: `RNA_input.tif` file
+>    - *"Histogram equalization algorithm"*: `CLAHE`
+> 2. Rename {% icon galaxy-pencil %} the generated file to `RNA_input_normalized`.
+> 3. Click on the **visualise icon** {% icon galaxy-visualise %} of the file to visually inspect the image using the **Tiff Viewer** visualization plugin.
+{: .hands_on}
+
+Your image should now look a bit brighter:
+
+![2025_post_clahe.png](../../images/2d_spot_detection/2025_post_clahe.png){: width="75%"}
+
+We can now clearly make out the presence of many RNA spots!
+
+# RNA spots detection
+
+We can now try to detect blobs and measure their intensity... Let's run a specific tool for this!
+
+> <hands-on-title>Perform 2-D spot detection</hands-on-title>
+>
+> 1. {% tool [Perform 2-D spot detection ](toolshed.g2.bx.psu.edu/repos/imgteam/spot_detection_2d/ip_spot_detection_2d/0.1+galaxy0) %} with the following parameters to detect the 2D spots:
+>    - {% icon param-file %} *"Intensity image or a stack of images"*: `RNA_input_normalized.tif` file
+>    - *"Starting time point (1 for the first frame of the stack) "*: `1`
+>    - *"Ending time point (0 for the last frame of the stack)"*: `0`
+>    - *"Detection filter"*: `Laplacian of Gaussian`
+>    - *"Minimum scale"*: `1`
+>    - *"Maximum scale"*: `2`
+>    - *"Minimum filter response (absolute)"*: `0.25`
+>    - *"Minimum filter response (relative)"*: `0`
+>    - *"Image boundary"*: `10`
+> 2. Rename {% icon galaxy-pencil %} the generated file to `RNA_spot_detected`.
+{: .hands_on}
+
+The tool produces a TSV file containing all detections, with the following columns:
+
+- **frame**: The frame of the image stack
+- **pos_x**: The horizontal coordinate of the detection
+- **pos_y**: The vertical coordinate of the detection
+- **scale**: The scale at which the detection was found
+- **radius**: The radius of the detected spot
+- **intensity**: The mean intensity of the spot
+
+The TSV output will have 1649 lines, meaning 1649 spots detected!
+
+![TSV_output.png](../../images/2d_spot_detection/TSV_output.png)
+
+# Convert spots coordinates to labels
+ Let's try now to visualize our results! The first step is to create a label map from the TSV file...
+This can be nicely done by converting the point coordinates to a label map and we have a tool for it!
+
+> <hands-on-title>Segment image</hands-on-title>
```

Comment:
Strictly speaking, segmentation is somewhat different than what happens here. Segmentation is like detection, but also determining the morphology and size of objects. This isn't the case here, right? Morphology is assumed to be circular.